zipline/finance/performance/tracker.py

```diff
@@ -296,6 +296,8 @@ def process_trade(self, event):
                 perf_period.handle_cash_payment(cash_adjustment)
 
     def process_transaction(self, event):
+        # Transaction events also count as trade events for the trackers
+        self.process_trade(event)
```

Comment:
> Would this create an unnecessary attribute access on process_trade that could impact perf (or is that negligible)?

I wouldn't expect attribute accesses on the perf tracker to have a noticeable impact.